README.md

```diff
@@ -1,123 +1,99 @@
-# Kubernetes Cluster API Azure Provider  [![Go Report Card](https://goreportcard.com/badge/sigs.k8s.io/cluster-api-provider-azure)](https://goreportcard.com/report/kubernetes-sigs/cluster-api-provider-azure)
+# Kubernetes Cluster API Provider Azure
 
-## Getting Started
+[![Go Report Card](https://goreportcard.com/badge/sigs.k8s.io/cluster-api-provider-azure)](https://goreportcard.com/report/sigs.k8s.io/cluster-api-provider-azure)
```

Comment:
```suggestion
[![Go Report Card](https://goreportcard.com/badge/kubernetes-sigs/cluster-api-provider-azure)](https://goreportcard.com/report/kubernetes-sigs/cluster-api-provider-azure)
```

Reply:
This doesn't need to be fixed. That's the correct import path.

Reply:
Unfortunately goreportcard requires the absolute links for the github permalinks inside of the lint reports to work.